### PR TITLE
feat : added lists

### PR DESCRIPTION
**Pull request by sach10-create**

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
looks good to merge, the UI also looks great just a suggestion that you can increase the width of sidebar

File: components/lists/src/css/stacked-list.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  gap: 1rem;
+  padding: 1rem;
+  background-color: var(--font-color);
+  color: var(--background-color);
+  border-radius: 1rem;
+}
+
+.stacked-list li {
+  margin: 2rem;
+}
```
Comment (zoef-yt) [inline components/lists/src/css/stacked-list.css:14]: Give spacing between line number 14 and 15.

File: css/stacked-list.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  gap: 1rem;
+  padding: 1rem;
+  background-color: var(--font-color);
+  color: var(--background-color);
+  border-radius: 1rem;
+}
+
+.stacked-list li {
+  margin: 2rem;
+}
```
Comment (zoef-yt) [inline css/stacked-list.css:14]: Added spacing.

**Review by zoef-yt — CHANGES_REQUESTED** [review]
Why there are so many duplicate files?

File: components/lists/src/css/stacked-list.css (hunk 1 above)
Comment (sach10-create) [reply components/lists/src/css/stacked-list.css:14]: Duplicates are there to maintain folder structure, they will be deleted once completed with the CL

**Pull request merged**